src/surge-xt/osc/OpenSoundControl.cpp

```diff
@@ -24,12 +24,14 @@
 #include "Parameter.h"
 #include "SurgeSynthProcessor.h"
 #include "SurgeStorage.h"
+#include <iostream>
 #include <sstream>
 #include <vector>
 #include <algorithm>
 #include <string>
 #include "UnitConversions.h"
 #include "Tunings.h"
+#include "filesystem/ghc-filesystem.h"
```

Comment:
this is not the right way to import filesystem. Instead do

```
#include "import/filesystem.h"
```

which will load one of `std::filesystem` or `ghc::filesystem` and set the namespace `fs` to point.

This is why your windows builds (where we don't use GHC) failed.

Reply:
Much appreciated, Paul. However, `#include "import/filesystem.h"` doesn't compile for me. I see SurgeStorage uses `#include "filesystem/import.h"` though, so probably that's the right header?

Reply:
yes! sorry about that

Reply:
Looks it still doesn't compile on windows. I'll investigate later, looks like I should convert Path to string before feeding to strncpy.